codeGenerator/configuration/configuration.xml

```diff
@@ -1,32 +1,21 @@
 <?xml version="1.0"?>
 <toolConfiguration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
-
-<rootOutputFolder>..\..\src\main\cpp</rootOutputFolder>
-<robotConfiguration>..\..\src\main\deploy\practiceBot.xml</robotConfiguration>
-
-<robotConfigurations>
-	<string>..\..\src\main\deploy\practiceBot.xml</string>
-	<string>..\..\src\main\deploy\robot.xml</string>
-</robotConfigurations>
-
-<treeviewParentNameExtensions>
-	<string>name</string>
-	<string>type</string>
-	<string>canId</string>
-	<string>pwmId</string>
-	<string>stateIdentifier</string>
-	<string>identifier</string>
-</treeviewParentNameExtensions>
-
-<GenerationNotice>
-//==============================================================
-// This file is auto generated by FRCrobotCodeGen302.exe Version $CODE_GENERATOR_VERSION$ 
-// Changes to this file may cause incorrect behavior and will be lost when 
-// the code is regenerated, unless the changes are delimited by:
-//==============================================================
-</GenerationNotice>
-
-<CopyrightNotice>
+  <rootOutputFolder>C:\FRC\2023SummerProject\src\main\cpp</rootOutputFolder>
```

Comment:
Why is this an absolute path to a local directory?

Reply:
not sure, I need to check